scripts/build/dependencies-npm.sh

```diff
@@ -101,7 +101,7 @@ function run_install() {
   # since compiled add-ons will not work otherwise.
   npm rebuild
 
-  npm install $INSTALL_OPTS
+  npm install $INSTALL_OPTS --fetch-retries 10 --fetch-retry-maxtimeout 180000
```

Comment:
Not that it matters, but would it have made more sense to make this change on line 85? i.e. adding to the INSTALL_OPTS variable?